src/maintainer/knowledge_base.rst

```diff
@@ -554,45 +554,70 @@ Special packages
 ================
 
 BLAS
-----
+....
 
-.. todo::
+If a package needs one of BLAS, CBLAS, LAPACK, LAPACKE, use the following in the
+host of the recipe,
+
+.. code-block:: yaml
+    requirements:
+      host:
+        - libblas
+        - libcblas
+        - liblapack
+        - liblapacke
+
+Note that you should specify only the libraries the package needs. (i.e. if the package
```

Comment:
In the parts I restructured I tried to consistently use `.. note:: ` directive, do you think this may be a good idea here, too?